test/fiddle/test_c_struct_entry.rb

```diff
@@ -49,6 +49,11 @@ def test_class_size_with_count
     end
 
     def test_set_ctypes
+      if RUBY_ENGINE == "jruby" and Fiddle::WINDOWS
+        omit("JRuby's 'l!' pack string is buggy. " +
+             "See https://github.com/jruby/jruby/issues/8357 for details")
```

Comment:
https://github.com/jruby/jruby/issues/8357

Reply:
@kou yeah for a while likely means JRuby 10 will fix this but 9.4 will continue to use that behavior since it would break any saved pack data for a JRuby app on windows.

Reply:
OK. I've added a workaround for JRuby 9.4 on Windows: https://github.com/ruby/fiddle/pull/147/commits/b8095834e1c1c3fa820bef2b8e020aa8f433dd91